ui/admin/app/templates/scopes/scope/session-recordings/session-recording/channels-by-connection/index.hbs

```diff
@@ -252,7 +252,16 @@
                       @model.sessionRecording.retain_until
                     }}
                   >
-                    {{format-date-iso @model.sessionRecording.retain_until}}
+                    {{#if
+                      (eq
+                        (format-date-iso @model.sessionRecording.retain_until)
+                        '9999-12-31T23:23:23.999Z'
```

Comment:
This is a constant from the API, open to suggestions if there's a better way to do this check.

Reply:
I didn't do this check in the controller because we get the format `Tue Fri 25 YYYY 00:00:00 GMT-0800 (Pacific Standard Time)`  and the value from the helper seemed like a better choice to do this check

Reply:
Could this be used elsewhere? If so I'd make it into a global helper otherwise I think this is fine

Reply:
no, this is the only place where we show `retain_until` date in the UI